app/Jobs/ImportFileRecords.php

```diff
@@ -53,11 +53,12 @@ class ImportFileRecords implements ShouldQueue
      * @param array $importOptions
      * @return void
      */
-    public function __construct($filepath, $importType, $importOptions)
+    public function __construct($user, $filepath, $importType, $importOptions)
```

Comment:
Just thought of this – can we add types here?

```suggestion
    public function __construct(\Chompy\User $user, string $filepath, string $importType, array $importOptions)
```

Reply:
We should also add this `$user` param to the docblock for this constructor too, and perhaps mention that it could be null for machine-triggered imports.